### DISCUSSION TITLE
Create Freeze Frame

### DISCUSSION THREAD

**Discussion by Randalix** [olive-editor/olive#2266] [Q&A] [answered]
Hey Hello,

I want to freeze the current frame of a clip in the timeline. Is there another way then setting to speed to 0.01, which is my current workaround?

**Comment by AngeloFrancabandiera** [comment, marked as answer]
Export the frame as a still image and import the exported frame in the timeline.
You have to set IN point in desired frame and OUT point in the next. Then choose PNG format in export dialog.

**Reply by mcDandy** [reply]
Too complicated. 
1. go to frame which you want to export
2. go to export dialog, set it to PNG and make sure image sequence export is not checked
3. below that setting is time which is set to current frame (the frame which will be exported)
4. set name and click export
